### PR TITLE
How to send message to Yandex Data Streams by raw HTTP

### PR DESCRIPTION
**Pull request by SloNN**
Python example how to send raw HTTP messages to Yandex Data Streams

### CODE REVIEW

File: yds/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Sample project for send messages by row HTTP.
+
+# Yandex Services
+* [Yandex Data Stream](https://cloud.yandex.ru/services/yds)
+* [Yandex Database](https://cloud.yandex.ru/services/ydb)
+
+# Prepare services
+
+1. Create _Yandex Data Streams_ stream and save full stream name
```
Comment (c4pQ) [inline yds/examples/README.md:11]: I'd add here what full stream name is

File: yds/examples/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+2. Create static credentials with `yds.writer` role
```
Comment (c4pQ) [inline yds/examples/README.md:12]: are you sure about the role?

File: yds/examples/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Example of how to send Yandex Data Streams messages by raw HTTP
+
+Sample project for send messages by row HTTP.
```
Comment (c4pQ) [inline yds/examples/README.md]: Sample project to send records within HTTP messages.

File: yds/examples/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Example of how to send Yandex Data Streams messages by raw HTTP
```
Comment (c4pQ) [inline yds/examples/README.md:1]: Example on

Comment (c4pQ) [reply yds/examples/README.md:1]: I'd also get rid of _raw_ adjective. kinesis records <-> HTTP messages

File: yds/examples/README.md (hunk 3 above)
Comment (SloNN) [reply yds/examples/README.md]: Fixed

File: yds/examples/README.md (hunk 4 above)
Comment (SloNN) [reply yds/examples/README.md]: Fixed

File: yds/examples/README.md (hunk 1 above)
Comment (SloNN) [reply yds/examples/README.md]: `Fixed

**Pull request merged**